src/storage-manager.coffee

```diff
@@ -67,6 +61,9 @@ class _StorageManager
     collection.on 'remove', (model) ->
       model.invalidateCache()
 
+    collection.on 'change', (model) =>
+      @_invalidateCache(model.brainstemKey) unless model.isNew()
```

Comment:
Nowadays, autocompleters tend to always bypass their cache because there are other UI parts which have the potential of changing the underlying data. It would be nice to only invalidate when those other controls actually change the data. 

I wanted to only invalidate when a model is saved to the server, but I was not sure if there is a specific event I could leverage. As far as I could tell, there is only "sync" which would be triggered when data is loaded -- forever bypassing the cache.